spec/ddtrace/contrib/action_cable/patcher_spec.rb

```diff
@@ -59,6 +59,10 @@
         expect(span.resource).to eq('action_cable.broadcast')
         expect(span.get_tag('action_cable.channel')).to eq(channel)
         expect(span.get_tag('action_cable.broadcast.coder')).to eq('ActiveSupport::JSON')
+        expect(span.get_tag(Datadog::Ext::Metadata::TAG_COMPONENT))
+          .to eq(Datadog::Contrib::ActionCable::Ext::TAG_COMPONENT)
+        expect(span.get_tag(Datadog::Ext::Metadata::TAG_OPERATION))
+          .to eq(Datadog::Contrib::ActionCable::Ext::TAG_OPERATION_BROADCAST)
```

Comment:
Minor inconsistency: On some of the tests we're comparing the two constants, and on some (below) we actually compare the actual strings.

I would actually suggest avoiding the constants in all cases, and hardcoding the strings in the tests, since it makes them more independent of the code (we can move the constants if needed), and it makes for a clear reading of the tests because you don't need to look up the contents of the tags -- it's right there.

Reply:
Totally agree, doing it now.